ui-admin/src/api/api.tsx

```diff
@@ -101,6 +101,21 @@ export type Enrollee = {
   profile: Profile
 }
 
+type RelationshipType = 'PROXY'
+
+export type EnrolleeRelation = {
+  id: string
+  relationshipType: RelationshipType,
+  targetEnrolleeId: string,
+  targetEnrollee: Enrollee
+  enrolleeId: string
+  enrollee: Enrollee
+  createdAt: number
+  lastUpdatedAt: number
+  beginDate: number
+  endDate: number
+}
```

Comment:
Similar types exist in `ui-participant`, should these go into `ui-core`?

Reply:
That's a good point- that same thing is getting in my way for the admin form completion work too, so good call to keep it out of this PR. You might get it for free in a few days anyway